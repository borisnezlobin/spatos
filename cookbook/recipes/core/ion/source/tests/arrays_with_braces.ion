echo [{ }]
echo [{a b c}]
echo [a{b c} d]
echo [a{b,c d} e]

echo {[,]}
echo {[a,b]}
echo {[a,b]c,{d,e}}
