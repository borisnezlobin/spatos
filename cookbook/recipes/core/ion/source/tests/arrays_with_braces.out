 
a b c
ab c d
ab ac d e
,
a,b
[a,b]c d e
